app/controllers/users_controller.rb

```diff
@@ -61,13 +57,11 @@ def destroy
 
   private
 
-  # Use callbacks to share common setup or constraints between actions.
   def set_user
     @user = User.find(params[:id])
   end
 
-  # Only allow a list of trusted parameters through.
   def user_params
-    params.require(:user).permit(:name, :surname, :nickname, :phone_number, :email, :birthday)
+    params.require(:user).permit(:name, :surname, :nickname, :phone_number, :email, :birthday, :role)
```

Comment:
careful! the role should be assigned by default and only the admin can modify it otherwise anyone could be admin and make a mess in the app